plugins/aws_sts/app/views/samson_aws_sts/_fields.html.erb

```diff
@@ -0,0 +1,45 @@
+<fieldset>
+  <legend>AWS STS Tokens</legend>
+  <div class="col-lg-offset-2">
+    <div class="form-group">
+      <div class="col-lg-6">
+        <p>Tokens will be requested using the following user: <%= SamsonAwsSts::Client.new.caller_user_id rescue "error connecting to aws, check SAMSON_STS_AWS env vars" %></p>
```

Comment:
```suggestion
        <p>Tokens will be requested using the following user: <%= SamsonAwsSts::Client.new.caller_user_id rescue "error connecting to aws, check SAMSON_STS_AWS_* env vars" %></p>
```

Reply:
might be nice to not cause an exception+rescue+error here when aws_sts is not configured

```
if client
  blah blah rescue error
else
  samson env vars are not configured
end
```